opentelemetry-instrumentation/src/opentelemetry/instrumentation/_semconv.py

```diff
@@ -310,27 +326,32 @@ def _set_http_target(result, target, path, query, sem_conv_opt_in_mode):
             set_string_attribute(result, URL_QUERY, query)
 
 
-def _set_http_peer_ip(result, ip, sem_conv_opt_in_mode):
+def _set_http_host_server(result, host, sem_conv_opt_in_mode):
     if _report_old(sem_conv_opt_in_mode):
-        set_string_attribute(result, SpanAttributes.NET_PEER_IP, ip)
+        set_string_attribute(result, SpanAttributes.HTTP_HOST, host)
     if _report_new(sem_conv_opt_in_mode):
-        set_string_attribute(result, CLIENT_ADDRESS, ip)
+        set_string_attribute(result, CLIENT_ADDRESS, host)
```

Comment:
This change wasn't too clear for me, but after taking a look [here](https://github.com/open-telemetry/semantic-conventions/blob/main/model/registry/deprecated/network.yaml) it makes sense.
>use `server.address` on client spans and `client.address` on server spans.